Fabric.Identity.API/Management/UserLoginManager.cs

```diff
@@ -113,7 +113,17 @@ private List<Claim> FilterClaims(IEnumerable<Claim> incomingClaims)
                 // copy the claim as-is
                 else
                 {
-                    filtered.Add(claim);
+                    if (claim.Type == FabricIdentityConstants.FabricClaimTypes.Groups)
+                    {
+                        if (string.IsNullOrEmpty(claim.Value) || !claim.Value.StartsWith("["))

```

Comment:
Do we want to add empty claims to the list?

Reply:
We were adding them in the original version.

```
else
{
   filtered.Add(claim);
}
```

Reply:
ok - just looked weird...what claims do we get that are empty?

Reply:
I don't see any in my DB that are null or empty, but the `UserClaims` table `Value` column is nullable.

Reply:
ok